app/components/schools/participants/coc_status_table.rb

```diff
@@ -29,10 +29,12 @@ def school_chose_cip?
         induction_records.first.school_cohort.core_induction_programme?
       end
 
+      # add the below back in once we do the transferring out journey
+      # transferring out currently list in the ECT or Mentor tabs
```

Comment:
minor - optional - you could make all the comments into #TODO comments which might help in finding them more easily after deployment

Reply:
I think something within RubyMine stops me from commit a TODO. i'll have a look at what is causing it